arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -2228,6 +2228,14 @@ void RocksDBEngine::getStatistics(VPackBuilder& builder) const {
     for (auto const& stat : rocksdb::TickersNameMap) {
       builder.add(stat.second, VPackValue(_options.statistics->getTickerCount(stat.first)));
     }
+
+    uint64_t wal_write, flush_write, compaction_write, user_write;
```

Comment:
according to our naming conventions, local variable names should always be camel-case.

Reply:
Agreed.  I had been reading rocksdb code all day.  Slipped into that naming style accidentally.  Will correct.

Reply:
done